extensions/pooling/README.md

```diff
@@ -103,7 +103,7 @@ const myPoolingService = await app.get<PoolingService>(
 ### Acquire a resource instance from the pool
 
 ```ts
-const res1 = await myPoolingService.acquire();
+const res1 = await myPoolingService.acquire(requestCtx);
```

Comment:
We should add a note about the optional nature and use case of passing a `requestCtx`.